web/src/compose_send_menu_popover.js

```diff
@@ -28,16 +29,30 @@ function set_compose_box_schedule(element) {
     return selected_send_at_time;
 }
 
-export function open_send_later_menu() {
-    if (!compose_validate.validate(true)) {
+export function open_send_later_menu(remind_message_id = undefined) {
+    if (remind_message_id === undefined && !compose_validate.validate(true)) {
         return;
     }
 
     // Only show send later options that are possible today.
     const date = new Date();
     const filtered_send_opts = scheduled_messages.get_filtered_send_opts(date);
-    $("body").append($(render_send_later_modal(filtered_send_opts)));
+    $("body").append(
+        $(
+            render_send_later_modal({
+                ...filtered_send_opts,
+                is_reminder: remind_message_id !== undefined,
+            }),
+        ),
+    );
     let interval;
+    const message_schedule_callback = (time) => {
+        if (remind_message_id === undefined) {
```

Comment:
I think doing `!== undefined` would be somewhat more readable -- checking something IS defined before using it is nicer than the way you've written it.

Reply:
Thanks, changed!